### DISCUSSION TITLE
Need a little help with my rad

### DISCUSSION THREAD

**Discussion by Kingagames77** [frntc/RAD#45] [Q&A]
Hello. Please I need help with my rad on pi zero 2. This is what happend after pressing reset or menu button.
![IMG_2120](https://github.com/frntc/RAD/assets/165667726/28dbd217-806b-4eb0-80ef-767a8cbd768c)
This is my rad
![Uploading IMG_2121.jpeg…]()

**Comment by frntc** [comment]
Hi, should there be a second photo in your post?

**Reply by Kingagames77** [reply]
Problems with upload. But my rad is ok. I’ve solder once again and it works. This is my first steps in smd soldering. Thanks for responce.
